src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -327,6 +335,8 @@ void ContextAttribute::present(const std::string& indent, int ix)
     compoundValueP->show(indent + "  ");
   }
 
+  PRINTF("%s  PA:       %s\n", indent.c_str(), providingApplication.c_str());
+
```

Comment:
Probably the `found` field should be also printed. Given that it is `bool` type, could you suggest a way of printint it? (if this were Python, simply `print str(found)` will work :)